app/src/parser.c

```diff
@@ -942,3 +950,34 @@ parser_error_t parser_getTxnText(parser_context_t *ctx,
 
     return parser_ok;
 }
+
+static parser_error_t validate_hd_path(const uint8_t *derivationPath) {
+    if (derivationPath == NULL) {
+        return parser_unexpected_error;
+    }
+
+    const char *expectedFormat = "m/44'/283'/";
```

Comment:
Shouldn't we take this from the HD path configuration we set on the makefile? Sounds like repeated values here